index.html

```diff
@@ -1,154 +1,184 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./index.css">
-    <link rel="stylesheet"
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./index.css">
+  <link rel="stylesheet"
     href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
-    <title>Amrita Map</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+  <title>Amrita Map</title>
 </head>
 
 <body>
 
-    <div class="desktopnav">
-        <div class="sandwichandscroller">
-            <button class="sandwich" aria-expanded="false">
-                <svg class="sandwichsvg" viewBox="0 0 20 20" width="22">
-                    <rect class="line top" width="20px" height="3px" y="3" rx="1.5"></rect>
-                    <rect class="line middle" width="20px" height="3px" y="9" rx="1.5"></rect>
-                    <rect class="line bottom" width="20px" height="3px" y="15" rx="1.5"></rect>
-                </svg>
-            </button>
-
-            <div class="mobile-nav">
-              <nav>
-                <a class="navbaractive" onclick="setTimeout(()=> {closenav()},200);">Map <span class="material-symbols-outlined leftmargin5">map</span></a>
-                <a href="#">Halls and Labs <span class="material-symbols-outlined leftmargin5">home_work</span></a>
-              </nav>
-            </div>
+  <div id="container">

```

Comment:
Did you make sure you didn't affect any existing element selectors in any of the script file by changing the hierarchy of the DOM tree by adding new <div></div>s with class names?